libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/events/FeedbackEvent.kt

```diff
@@ -7,105 +7,105 @@ import androidx.annotation.StringDef
+     */
+    const val ACCIDENT = "accident"
+
+    /**
+     * Feedback type *hazard*: such as debris, stopped vehicles, etc
+     */
+    const val HAZARD = "hazard"
+
+    /**
+     * Feedback type *road closed*: closed road or one that does not allow vehicles
+     */
+    const val ROAD_CLOSED = "road_closed"
+
+    /**
+     * Feedback type *not allowed*: a turn/maneuver that isn't allowed
+     */
+    const val NOT_ALLOWED = "not_allowed"
+
+    /**
+     * Feedback type *routing error*: poor instruction or route choice
+     * (ambiguous or poorly-timed turn announcement, or a set of confusing turns)
+     */
+    const val ROUTING_ERROR = "routing_error"
+
+    /**
+     * Feedback type *missing road*: road doesn't exist
+     */
+    const val MISSING_ROAD = "missing_road"
+
+    /**
+     * Feedback type *missing exit*: dead end
+     */
+    const val MISSING_EXIT = "missing_exit"
+
+    /**
+     * Feedback type *confusing instruction*: wrong voice/text instruction
+     */
+    const val CONFUSING_INSTRUCTION = "confusing_instruction"
+
+    /**
+     * Feedback type *inaccurate gps*: wrong location of navigation puck on the map
+     */
+    const val INACCURATE_GPS = "inaccurate_gps"
+
+    /**
+     * Feedback type *incorrect visual guidance*: wrong visual guidance
+     */
+    const val INCORRECT_VISUAL_GUIDANCE = "incorrect_visual_guidance"
+
+    /**
+     * Feedback type *incorrect audio guidance*: wrong audio guidance
+     */
+    const val INCORRECT_AUDIO_GUIDANCE = "incorrect_audio_guidance"
+
+    /**
+     * Feedback source *reroute*: the user tapped a feedback button in response to a reroute
+     */
+    const val REROUTE = "reroute"
+
+    /**
+     * Feedback source *user*: the user tapped a feedback button
+     */
+    const val UI = "user"
 
     /**
      * Type of feedback mean WHAT happen
      */
     @Retention(AnnotationRetention.SOURCE)
     @StringDef(
-        FEEDBACK_TYPE_GENERAL_ISSUE,
-        FEEDBACK_TYPE_ACCIDENT,
-        FEEDBACK_TYPE_HAZARD,
-        FEEDBACK_TYPE_ROAD_CLOSED,
-        FEEDBACK_TYPE_NOT_ALLOWED,
-        FEEDBACK_TYPE_ROUTING_ERROR,
-        FEEDBACK_TYPE_CONFUSING_INSTRUCTION,
-        FEEDBACK_TYPE_INACCURATE_GPS,
-        FEEDBACK_TYPE_MISSING_ROAD,
-        FEEDBACK_TYPE_MISSING_EXIT
+        GENERAL_ISSUE,
+        ACCIDENT,
+        HAZARD,
+        ROAD_CLOSED,
+        NOT_ALLOWED,
+        ROUTING_ERROR,
+        MISSING_ROAD,
+        MISSING_EXIT,
+        CONFUSING_INSTRUCTION,
+        INACCURATE_GPS,
+        INCORRECT_VISUAL_GUIDANCE,
+        INCORRECT_AUDIO_GUIDANCE
     )
-    annotation class FeedbackType
+    annotation class Type
 
     /**
      * Feedback source mean WHERE happen
      */
     @Retention(AnnotationRetention.SOURCE)
     @StringDef(
-        FEEDBACK_SOURCE_REROUTE,
-        FEEDBACK_SOURCE_UI
+        REROUTE,
+        UI
     )
-    annotation class FeedbackSource
+    annotation class Source
```

Comment:
out of scope of a package change: https://github.com/mapbox/mapbox-navigation-android/pull/2839/files#r416766318

Reply:
but also my preference is FeedbackSource. There's already a few "Source" objects

![Screen Shot 2020-04-28 at 9 52 27 AM](https://user-images.githubusercontent.com/3021882/80514889-faaaf000-8935-11ea-88a3-74f1dc1909c3.png)